# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/AudioStream.h
# Commit: 3fbc1456c6d2
# Full Hash: 3fbc1456c6d26d91729498ea6ad19dc24e479dfa
# Author: Matthew Gregan <kinetik@flim.org>
# Date: 2019-10-02 03:38:52
# Regressor Bug: 1544997
# File Overlap Count: 1
# Description:
#   Bug 1544997 - Propogate cubeb stream error to AudioSink to handle loss of audio device.  r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D47745
# ==============================================================================

diff -r f53153babb6f -r 3fbc1456c6d2 dom/media/AudioStream.h
--- a/dom/media/AudioStream.h	Tue Oct 01 20:01:54 2019 +0000
+++ b/dom/media/AudioStream.h	Tue Oct 01 12:14:44 2019 +0000
@@ -201,6 +201,8 @@
     virtual bool Ended() const = 0;
     // Notify that all data is drained by the AudioStream.
     virtual void Drained() = 0;
+    // Notify that a fatal error has occured during playback.
+    virtual void Errored() = 0;
 
    protected:
     virtual ~DataSource() = default;